digitalforms-api/src/main/java/ca/bc/gov/open/pssg/rsbc/digitalforms/controller/IRPReviewController.java

```diff
@@ -48,12 +48,12 @@
 	}
 
 	@PatchMapping(value = "/{irpNoticeNumber}/application/{id}")
-	public ResponseEntity<JSONResponse<String>> irpReviewPatch(
+	public ResponseEntity<JSONResponse<String>> irpReviewFormPatch(
 			@PathVariable(value = "irpNoticeNumber", required = true) Long irpNoticeNumber,
 			@PathVariable(value = "id", required = true) Long id, @RequestBody IRPReviewFormRequest formData) {
 		String data = irpService.patchIRPReviewForm(irpNoticeNumber, id, formData);
 		JSONResponse<String> resp = new JSONResponse<>(data);
-		return new ResponseEntity<>(resp, HttpStatus.NO_CONTENT);
+		return new ResponseEntity<>(resp, HttpStatus.OK);
```

Comment:
Http 204 (No_content) is not bringing back any JSON response. Hence changing it back to http 200